tensorflow/lite/kernels/arg_min_max_test.cc

```diff
@@ -152,6 +148,38 @@ TEST(ArgMaxOpTest, GetMaxArgOutput64) {
   EXPECT_THAT(model.GetOutputShape(), ElementsAreArray({1, 1, 2}));
 }
 
+TEST(ArgMaxOpTest, GetMaxArgAxis64) {
```

Comment:
nit: Is

Reply:
@renjie-liu I didn't get it. Can you elaborate? TIA.

Reply:
sorry my bad, miss read it